torch/testing/_internal/common_distributed.py

```diff
@@ -180,7 +180,8 @@ def skip_if_lt_x_gpu(x):
     def decorator(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
-            if torch.cuda.is_available() and torch.cuda.device_count() >= x:
+            if (torch.cuda.is_available() and torch.cuda.device_count() >= x) or \
+               (torch.xpu.is_available() and torch.xpu.device_count() >= x):
```

Comment:
Should we provide comprehensive support for the testing? Why does `at_least_x_gpu` not need to support Intel GPU? Because we do not test it?

Reply:
Yes, we will provide sufficient tests. This modification is limited to this particular section because the test cases for this PR currently only involve this part. Changes to unit tests will be coordinated with the frontend PRs (e.g., DDP, FSDP, etc.).